moveit_planners/chomp/chomp_interface/chomp_interface_plugin_description.xml

```diff
@@ -0,0 +1,6 @@
+<library path="lib/libchomp_planner_plugin">
+  <class name="chomp_interface/CHOMPPlanner" type="chomp_interface::CHOMPPlannerManager" base_class_type="planning_interface::PlannerManager">
+    <description>
+    </description>
```

Comment:
better yet - "CHOMP: Gradient Optimization Techniques for Efficient Motion Planning"